lib/encryption/encrypt.js

```diff
@@ -0,0 +1,11 @@
+const bcrypt = require('bcrypt')
+const SALT_ROUNDS = 10
+
+async function encryptString (value) {
+  const encryptedString = await bcrypt.hash(value, SALT_ROUNDS)
```

Comment:
async methods are preferred in Node.js over sync.  The way Node's event loop works it's bad practice to block where asynchronous calls can be made.  Here's an overview (https://blog.logrocket.com/a-complete-guide-to-the-node-js-event-loop/)